### PR TITLE
fix(github): Error messages for empty repository result

### PR DESCRIPTION
**Pull request by zharinov**
## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [ ] Code inspection only, or
- [x] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

**Review by rarkins — COMMENTED** [review]
What's the expected result now if res.repository is null?

**Comment by zharinov** [comment]
Still error, but more informative

**Comment by zharinov** [comment]
But if there is anything in `errors`, now it will throw with the error message, not the null pointer exception

File: lib/modules/datasource/github-releases/cache/cache-base.ts [hunk 1]
```diff
@@ -179,7 +179,22 @@ export abstract class AbstractGithubDatasourceCache<
       });
       const { body } = graphqlRes;
       const { data, errors } = body;
-      return data ?? new Error(errors?.[0]?.message);
+
+      if (errors) {
+        let [errorMessage] = errors
+          .map(({ message }) => message)
+          .filter(Boolean);
```
Comment (viceice) [inline lib/modules/datasource/github-releases/cache/cache-base.ts:186]:
```suggestion
          .filter(is.string);
```

**Review by viceice — APPROVED** [review]

**Pull request merged**